how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -160,7 +138,9 @@
     "    capture_hardware_metrics=True,\n",
     "    capture_stderr=True,\n",
     "    capture_stdout=True,\n",
-    ")  # Hardware metrics, stderr, and stdout are not captured by default in interactive kernels"
+    "    source_files=\"hello_neptune.ipynb\",  # the entrypoint .py file is logged automatically\n",
```

Comment:
ah yes, this does not work in Colab.
Since passing a notebook here was already kind of a workaround, I think we should just skip source_files tracking in the quickstart notebook.
@parthpankajtiwary - thoughts?

Reply:
If it doesn't upload the `hello_neptune.ipynb` then we can leave it out. 

Alternatively, although it may be considered an anti-pattern, is to initialize the `source_files` parameter as an empty list `source_files=[]` and inform users that they have the option to upload specific files if needed (some redirection to the docs). By doing this, users won't encounter any immediate roadblocks but will have the flexibility to explore the concept further if they need it.

What do you think? @SiddhantSadangi @normandy7

Reply:
Fine by me. I think it's worth showing the `source_files` arg, as setting it to an empty list doesn't "cost" anything but does highlight its existence.

We may still want to mention the Colab limitation in the notebook itself, otherwise pointing to the docs should be enough. https://docs.neptune.ai/logging/source_code

Reply:
Keeping `source_files` as empty might lead to nothing being tracked if the users download the notebook to run locally without updating `source_files`.
I think a better option would be to comment out `source_files="hello_neptune.ipynb"` and direct users to the docs on shource_files behavior. This will ensure default `source_files` behavior even if run locally, and provide users with a quick way of uploading notebooks if they want to.

```python
# source_files="hello_neptune.ipynb", # Neptune logs certain source files automatically. Read https://docs.neptune.ai/logging/source_code/ for details and caveats
```

Reply:
But this doesn't make it immediately apparent why the line is commented or in what circumstance it could be uncommented for the intended effect. The original version would log a locally executed notebook out of the box (provided the file name wasn't changed on download).

I'd stick with the Colab comment in the code cell so that it's almost unmissable why the notebook isn't uploaded in that case. This way the docs link would be extra and not mandatory for understanding what's happening.

Reply:
So just the colab comment, or that with the docs link?
I was just concerned about the length of the comment.

I am fine with just the colab comment as well

Reply:
I have no strong preferences as long as we can demonstrate the `source_files` functionality while mitigating unexpected behaviour and confusion in the final proposal. _Commenting / uncommenting_ doesn't sound clean, I will be inclined for alternatives.

Reply:
How about this?
```
source_files="hello_neptune.ipynb", # works in locally executed notebooks (not in Colab). Read https://docs.neptune.ai/logging/source_code/ for details and caveats
```

Reply:
Does the whole thing have to be in the code comment? I'd try moving the link to a text cell and phrase it as "For how to upload source files from within a notebook, see [Logging source code](https://docs.neptune.ai/logging/source_code) in the Neptune docs."

Reply:
On second thoughts (sorry for the to and fro), I think Sabine's initial suggestion makes the most sense. We do not need to mention all the caveats in the quickstart.

I'd only update the comment to "# does not work in Colab" as it works on some other non-local environments